### DISCUSSION TITLE
Tabs Storage Limit

### DISCUSSION THREAD

**Discussion by ShaSha146** [XFox111/TabsAsideExtension#85] [Q&A]
There appears to be a limit on the number of tabs that can be set aside at once or in total. Can this limit be changed or is it caused by a limit in the Extension API. When using the collections feature in Edge there doesn't appear to be a limit on the number of tabs that can be stored.

**Comment by XFox111** [comment]
Hi, thanks for your question!

The limit was introduced when we added a cross-device synchronization of sets. Unfortunately, Chromium provides very little space to store user's data. Probably, in future we will make an option to turn off device sync, thus expand available storage

**Reply by AmineI** [reply]
For now @ShaSha146 , a workaround could be to just clone the repo and edit this to false https://github.com/XFox111/TabsAsideExtension/blob/9c734f7495c25fd2a8670f58ede99993041e28f1/js/background.js#L5
before loading the unpacked extension in your browser. 

This can be done by following these steps : 
> Enable Developers mode on your browser extensions page
> Click "Load unpacked" button and navigate to the extension root folder (contains manifest.json)

Of course, syncing your tabs set aside is not possible anymore if you disable that.
